### DISCUSSION TITLE
Regarding the project HTML files under static folder

### DISCUSSION THREAD

**Discussion by nk8291** [asreview/asreview#1207] [Q&A] [answered]
How do we can get the HTML files used in the front end of the application. Like html codes for project creation, add/upload files/ get URL etc..

**Comment by J535D165** [comment, marked as answer]
ASReview LAB is written in React. All components and HTML code are available in the JavaScript files (`.js`).

**Reply by Rensvandeschoot** [reply]
See for example the files used for the project set-up phase: https://github.com/asreview/asreview/tree/master/asreview/webapp/src/ProjectComponents

**Reply by nk8291** [reply]
Thank you for the quick response :)
